packages/fast-data-utilities-react/src/index.ts

```diff
@@ -0,0 +1,457 @@
+import * as React from "react";
+import { cloneDeep, get, isPlainObject, set } from "lodash-es";
+import * as tv4 from "tv4";
+
+export interface ChildOptionItem {
+    /**
+     * The name of the component
+     */
+    name?: string;
+
+    /**
+     * The React component
+     */
+    component: React.ComponentClass;
+
+    /**
+     * The JSON schema for the component
+     */
+    schema: any;
```

Comment:
`scheme: JSON` - JSON is a type now

Reply:
unless it makes you refactor everything - then you we should just do a follow-up